diff --git a/sshd-auth.c b/sshd-auth.c
index 76350a2..08543e9 100644
--- a/sshd-auth.c
+++ b/sshd-auth.c
@@ -658,9 +658,11 @@ main(int ac, char **av)
 			fatal("Privilege separation user %s does not exist",
 			    SSH_PRIVSEP_USER);
 	} else {
+#ifndef __MVS__
 		privsep_pw = pwcopy(privsep_pw);
 		freezero(privsep_pw->pw_passwd, strlen(privsep_pw->pw_passwd));
 		privsep_pw->pw_passwd = xstrdup("*");
+#endif
 	}
 	endpwent();
 
